src/components/UI/molecules/Tab/Tab.interface.tsx

```diff
@@ -0,0 +1,9 @@
+import { ITabButton } from '../TabButton'
+
+export interface ITab extends Pick<ITabButton, 'link' | 'text' | 'size'> {
```

Comment:
no property from ITabButton is used in Tab component

Reply:
Type removed from interface